python/pyspark/ml/classification.py

```diff
@@ -65,17 +66,6 @@ class LogisticRegression(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredicti
     """
 
     # a placeholder to make it appear in the generated doc
-    elasticNetParam = \
-        Param(Params._dummy(), "elasticNetParam",
-              "the ElasticNet mixing parameter, in range [0, 1]. For alpha = 0, " +
-              "the penalty is an L2 penalty. For alpha = 1, it is an L1 penalty.")
-    fitIntercept = Param(Params._dummy(), "fitIntercept", "whether to fit an intercept term.")
-    thresholds = Param(Params._dummy(), "thresholds",
-                       "Thresholds in multi-class classification" +
-                       " to adjust the probability of predicting each class." +
-                       " Array must have length equal to the number of classes, with values >= 0." +
-                       " The class with largest value p/t is predicted, where p is the original" +
-                       " probability of that class and t is the class' threshold.")
     threshold = Param(Params._dummy(), "threshold",
```

Comment:
My understanding is that the `HasThresholds` trait mixin in `ml.LogisticRegression` is actually an artifact resulting from transient dependency through `ProbabilisticClassifier`. We don't actually support multi-class classification in `ml.LogisticRegression` ATM and did quite a bit of work to make the API less confusing.

After mutli-class is supported I think it makes sense to use `HasThresholds`, but for the time being I would prefer we only use `HasThreshold` in the Python API.